pydantic/errors.py

```diff
@@ -341,3 +341,7 @@ class ColorError(PydanticValueError):
 
 class StrictBoolError(PydanticValueError):
     msg_template = 'value is not a valid boolean'
+
+
+class LiteralError(PydanticValueError):
```

Comment:
can we use the same error as for `Schema(..., const=True)`. I don't mind if that error message has to change, but it's code should probably stay the same if possible.